server_addon/maya/server/settings/publishers.py

```diff
@@ -1371,6 +1600,52 @@ class PublishersModel(BaseSettingsModel):
     "ExtractModel": {
         "enabled": True,
         "optional": True,
-        "active": True,
+        "active": True
+    },
+    "ExtractAlembic": {
+        "enabled": True,
+        "families": [
+            "pointcache",
+            "model",
+            "vrayproxy.alembic"
+        ],
+        "attr": "",
+        "attrPrefix": "",
+        "autoSubd": False,
+        "bake_attributes": [],
+        "bake_attribute_prefixes": [],
+        "dataFormat": "ogawa",
+        "eulerFilter": False,
+        "melPerFrameCallback": "",
+        "melPostJobCallback": "",
+        "overrides": [
+            "attr",
+            "attrPrefix",
+            "renderableOnly",
+            "visibleOnly",
+            "worldSpace",
+            "writeColorSets",
+            "writeFaceSets",
+            "writeNormals"
+        ],
+        "preRoll": False,
+        "preRollStartFrame": 0,
+        "pythonPerFrameCallback": "",
+        "pythonPostJobCallback": "",
+        "renderableOnly": False,
+        "stripNamespaces": True,
+        "uvsOnly": False,
+        "uvWrite": False,
```

Comment:
Do we really feel this is a sensible studio default we should offer from AYON by default. With this `False` by default Alembics will be written **without UVs** - pretty sure we'll want to flip that. Right @tokejepsen @mkolar ?